autoload/go/package.vim

```diff
@@ -163,7 +175,11 @@ function! go#package#FromPath(arg) abort
     return -1
   endif
 
-  let l:importpath = split(l:out, '\n')[0]
+  let l:lines = split(l:out, '\n')
+  if empty(l:lines)
+    return -1
+  endif
+  let l:importpath = l:lines[0]
```

Comment:
How can `go list` with no arguments exit 0 _and_ not have any lines in its output?